package.json

```diff
@@ -67,21 +64,24 @@
     "eslint-plugin-react": "7.4.0",
     "jest": "^21.2.1",
     "jest-enzyme": "^4.0.1",
-    "node-sass-chokidar": "^0.0.3",
+    "node-sass": "^4.7.2",
```

Comment:
chokidar seems more performant, and is part of CRA recommendation to setup Sass
But we can go back to "node-sass", and change to chokidar later, I see value too on consuming the original library too, until we find a good reason. It would be an easy switch

Reply:
Yes, chokidar is part of CRA recommendation.  I went with standard node-sass as it came with the webpack-plugin... agreed, is an easy switch later 😊